Source/Core/Core/FifoPlayer/FifoPlayer.cpp

```diff
@@ -168,6 +168,17 @@ std::unique_ptr<CPUCoreBase> FifoPlayer::GetCPUCore()
   return std::make_unique<CPUCore>(this);
 }
 
+void FifoPlayer::SetFileLoadedCallback(CallbackFunc callback)
+{
+  m_FileLoadedCb = callback;
```

Comment:
This is pre-existing, so obvious non-blocker, but,

```suggestion
  m_FileLoadedCb = std::move(callback);
```

would be preferable here, since std::function is allowed to allocate to hold any captures that exceed its internal buffer, so this can allow avoiding reallocations.

Reply:
I'll do the same to both callbacks.